interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -373,6 +374,10 @@ protected ManagedChannel createChannel() {
           && customCredentialsType.equals("google_default_credentials")) {
         return GoogleDefaultChannelBuilder.forAddress(serverHost, serverPort).build();
       }
+      if (customCredentialsType != null
+          && customCredentialsType.equals("compute_engine_credentials")) {
```

Comment:
sorry, mind changing to `compute_engine_channel_creds` ?

`compute_engine_credentials` is very similar to the `compute_engine_creds` test case, and it also may look like it corresponds to compute engine "call creds".

Reply:
Done.